charts/kong/README.md

```diff
@@ -712,7 +712,7 @@ section of `values.yaml` file:
 #### The `env` section
 For a complete list of all configuration values you can set in the
 `env` section, please read the Kong Ingress Controller's
-[configuration document](https://github.com/Kong/kubernetes-ingress-controller/blob/main/docs/references/cli-arguments.md).
+[configuration document](https://github.com/Kong/docs.konghq.com/blob/main/src/kubernetes-ingress-controller/references/cli-arguments.md).
```

Comment:
I wonder if it wouldn't be better to link to https://docs.konghq.com/kubernetes-ingress-controller/latest/references/cli-arguments/ instead 🤔

Reply:
I considered that too but the "official docs" are autogenerated from the markdown itself (right?)

Reply:
That's what I gathered from [Kong/docs.konghq.com](https://github.com/Kong/docs.konghq.com)

Reply:
True. Actually, yes, let's keep the reader in this repo (do not link outside). 👍